lib/interactor.rb

```diff
@@ -163,4 +163,30 @@ def call
   # Returns nothing.
   def rollback
   end
+
+  private
+
+  # Internal: Determine what arguments (if any) should be passed to the "call"
+  # instance method when invoking an Interactor. The "call" instance method may
+  # accept any combination of positional and keyword arguments. This method
+  # will extract values from the context in order to populate those arguments
+  # based on their names.
+  #
+  # Returns an Array of arguments to be applied as an argument list.
+  def arguments_for_call # rubocop:disable Metrics/MethodLength
```

Comment:
I think I'll end up backing out the positional argument support. It feels a little _too_ magical in hindsight. Plus, I think there's a bug that I haven't tested for yet that could misplace positional arguments. The case I'm wondering about is:

```ruby
class MyInteractor
  include Interactor

  def call(foo, hello = "world")
    context.output = [foo, hello]
  end
end

result = MyInteractor.call(hello: "Anton")
result.output # This should raise an ArgumentError, but might be ["Anton", "world"]
```